packages/design-tokens/themes/light.ts

```diff
@@ -105,6 +105,10 @@ const misc = {
   danger: '#D23F47',
   success: '#138849',
   warning: '#F5C625',
```

Comment:
Remember to mark these as deprecated

Reply:
Done in [6f6a687](https://github.com/sumup-oss/circuit-ui/pull/1431/commits/6f6a687dea9e1d2a955b6c9d428939ffadff9714). Let me know if there is something else needed.